index.html

```diff
@@ -4,17 +4,227 @@
+                    alt="personal-picture">
+            </div>
+        </div>
+    </div>  
+
+<!-- Therapy Option Boxes -->
+    <div class="container">
+        <div class="row" style="text-align:center; margin-top: 450px; padding-bottom: 100px;">
+            
+            <div class="col-md-4" style="background-color: #E2F6CD; padding: 100px 5px;">
+                <p><i>01</i></p>
+                <h1>Individual Therapy</h1>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa.</p>
+            </div>
+        
+            <div class="col-md-4" style="background-color: #F2F2F1; padding: 100px 5px;">
+                <p><i>02</i></p>
+                <h1>Group Therapy</h1>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa.</p>
+            </div>
+            <div class="col-md-4" style="background-color: #CFF4EC; padding: 100px 5px;">
+                <p><i>03</i></p>
+                <h1>Couple Therapy</h1>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa.</p>
+            </div>
+        </div>
+    </div>
+
+<!-- Article Boxes -->
+    <div class="container" style="background-color:#F2F2F1; padding-bottom: 150px;">
+        <div class="row" style="padding-top: 50px; "> 
+            <div class="col">
+                <h5 class="text-center"><i>Articles About Psychology</i></h5>
+                    <div class="container">
+                        <div class="row">
+                            <!-- <div class="col">
+                                <div class="container">
+                                    <div class="row" > -->
+
+                                    
+                                        <div class="col-10 offset-1" style="background-color: white; text-align: center; padding-top: 80px;">
+                                            <div class="row justify-content-around">
+                                            <div class="col-3">
+                                                <a href="#"><img src="./css/succulent - article resize3.png"></a>
+                                                <p><i>Understanding PTSD</i></p>
+                                                <form style="d-flex">
+                                                    <button class="btn btn-outline-success" type="submit">Write a comment</button>
+                                                </form>
+                                            </div>
+                                       
+                                   
+                                        
+                                    
+                                       
+                                            <div class="col-3" >
+                                                <img src="./css/fern - article resize3.png" a href="#"></a>
```

Comment:
These images are breaking the layout on mobile. Can you add some code to stack these articles on smaller screens?